README.md

```diff
@@ -62,6 +62,7 @@ The function return Promise.
 ##### Options
 
 Options currently supports:
+- `framework` - Defines in which framework the middleware is working ('koa' or 'express'). As default, set to 'express'.
```

Comment:
Doesn't look natural to handle this using an option, can we export two different wrappers instead so that user could import whichever he needs?

Reply:
@kibertoad I tried to think on a better solution but with the current architecture of the package, I wasn't able to think of a better one.
I'm going to except it with the option parameter and we might refactor that later on.